src/triggers/workNotification.trigger

```diff
@@ -0,0 +1,9 @@
+trigger workNotification on Work_Item__c (after insert) {
+    for(Work_Item__c w : Trigger.New){
+        String workItemURL = URL.getSalesforceBaseUrl().toExternalForm() +'/'+ w.id;
+        Contact c = [SELECT Email FROM Contact WHERE id =:w.Assignee__c LIMIT 1];
```

Comment:
It's not actually a good idea to add SELECT statements inside loops because you might run into governor limits. To do this more efficiently you need to create a list of Id, loop through all items in Trigger.New and add the item's Assignee__c field to the list. 
After that, do a select on the Contact object where the id is included in those in your list. 
Afterwards, you need to loop through the contacts and send the respective emails.